Expensify/App
# Fix android copy paste menu doens't show up in compose box

### Details

Fix copy paste context menu won't show up in android.
Change `TextInput` of compose box into uncontrolled `TextInput` that will improve performance of `TextInput`

### Fixed Issues

$ https://github.com/Expensify/App/issues/6876

### Tests

1. Type something in compose box in Android
2. Select/highlight part of text 
3. Verify that Copy / Paste menu context show up in Android
4. Set selection/cursor at certain part of text 
5. Input an emoji
6. Verify emoji placement, cursor position and text change is correct after input an emoji

### QA Steps


1. Type something in compose box in Android
2. Select/highlight part of text 
3. Verify that Copy / Paste menu context show up in Android
4. Set selection/cursor at certain part of text 
5. Input an emoji
6. Verify emoji placement, cursor position and text change is correct after input an emoji

### Tested On


### Screenshots


#### Web


https://user-images.githubusercontent.com/1313124/153714274-f5061bd8-44a9-410f-9515-79c71b74a8aa.mp4


#### Mobile Web


https://user-images.githubusercontent.com/1313124/153714681-c847d869-6f94-4b0e-9cec-1148f9abe89e.mp4


#### Desktop


https://user-images.githubusercontent.com/1313124/154658222-90880932-b71a-4e40-871a-7e0fd51d7ee8.mp4


#### iOS


https://user-images.githubusercontent.com/1313124/154636904-d2175507-edcb-4327-b853-9f7ba679cf16.mp4


#### Android


https://user-images.githubusercontent.com/1313124/153714288-f30483e7-9ca3-484c-94ea-1a2a8580509b.mp4
